### PR TITLE
Use empty state component for empty snapshot list WD-2127

### PR DESCRIPTION
**Pull request by edlerd**
* changed the empty state for snapshots (added a create button and aligned the visual appearance)

### CODE REVIEW

File: static/js/pages/instances/InstanceSnapshots.tsx [hunk 1]
```diff
@@ -83,45 +87,40 @@ const InstanceSnapshots: FC<Props> = ({ controlTarget, instance, notify }) => {
...
+            rows={rows}
+            paginate={30}
+            responsive
+            sortable
+            className="u-table-layout--auto"
+          />
+        ) : (
+          <EmptyState
+            iconName="settings"
+            iconClass="p-empty-instances"
```
Comment (lorumic) [inline static/js/pages/instances/InstanceSnapshots.tsx:114]: In order to apply the correct colour, this should be changed to some new class called `p-empty-snapshots` which includes the mixin of the correct (temporary) icon:
```
.p-empty-snapshots {
  @include vf-icon-settings($color-mid-light);
}
```

Comment (edlerd) [reply static/js/pages/instances/InstanceSnapshots.tsx:114]: I changed it [here](https://github.com/canonical/lxd-ui/pull/213/files#diff-f5a5b0429dba7d86dd96396c39539ab3a091d90536bccb9db008a3dbc487f83dR102). Will pick that change to this branch, to allow reusing of the icon class.

Comment (lorumic) [reply static/js/pages/instances/InstanceSnapshots.tsx]: This is a different thing: not the icon for a link, but the icon representing the snapshots - as it was `vf-icon-containers` for instances, it is `vf-icon-settings` (temporarily) for snapshots.
It is to this icon that the correct colour (`$color-mid-light`) should be applied.

Comment (edlerd) [reply static/js/pages/instances/InstanceSnapshots.tsx]: Right, I mixed up the external link icon and the snapshot icon. Now it should be fixed.

**Review by lorumic — APPROVED** [review]
LGTM

**Pull request merged**